kajas/backend/controllers/mailer.js

```diff
@@ -0,0 +1,100 @@
+const { log } = require('@angular-devkit/build-angular/src/builders/ssr-dev-server');
+const nodemailer = require('nodemailer');
+const crypto = require('crypto');
+const bcrypt = require('bcryptjs');
+const User = require('../models/User');
+
+
+const transporter = nodemailer.createTransport({
+    host: 'smtp.hostinger.com',
+    port: 465,
+    secure: true, 
+    auth: {
+        user: 'midknightv03@kajas.site',
+        pass: 'Kajas_123'
+    }
+});
+
+const mailer =  async (req, res) => {
+    try {
+        const { email } = req.body
+        console.log(email);
```

Comment:
### Remove console.logs() and unneccessary codes in the production environment.
- Use console.error() for logging error messages.
- Remove codes/variables that are not used.
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g2637aafa621_0_0